### DISCUSSION TITLE
Installing and initializing buildozer utility in Visual Studio Code Terminal

### DISCUSSION THREAD

**Discussion by JohnCTXP** [microsoft/vscode-discussions#2425] [Extension Development QnA] [answered]
I am having problems with using the buildozer utility library in Visual Studio Code Terminal.

Here I have this error:
`buildozer : The term 'buildozer' is not recognized as the name of a cmdlet, function, script file, or      
operable program. Check the spelling of the name, or if a path was included, verify that the path is       
correct and try again.
At line:1 char:1
+ buildozer init
+ ~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (buildozer:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException`

The error log describes that buildozer is not a proper command.

My purpose is to import it to an APK or android package file, so that I can sync or side load it to one of my Android operating system embedded tablets.

Can anyone assist me in how to do this task?

**Comment by isidorn** [comment, marked as answer]
This is a good question for stack overflow.
This forum is for VS Code extension authors.
